ffmpeg/README.md

```diff
@@ -1,36 +1,38 @@
-# JavaCPP Presets for FFmpeg
+JavaCPP Presets for FFmpeg
+==========================
```

Comment:
I would ask that these markdown changes not be reverted. They show up as red squiggle errors in my editor (i.e. fail markdown linting).

Reply:
There's no reason to make it inconsistent with the other README.md files. If there's a good reason to change it though, we'll do it for all of them, and that's going to be for another pull request if you want to do that. However, maybe it's just your linting tool that's buggy. How do you know that's not the case?

Reply:
I think the tool is reporting valid violations of these rules:

https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md014---dollar-signs-used-before-commands-without-showing-output

https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md006---consider-starting-bulleted-lists-at-the-beginning-of-the-line

https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md003---header-style

I accept that could be a different commit.

Reply:
Yeah, let's do that in different commits, for the other repositories too!

BTW, it doesn't appear to violate setext_with_atx, that looks alright. The reason I do it that way is that it looks better with setext, but there aren't any for level 3. I guess I'm not the only one to feel that way.